### DISCUSSION TITLE
Change-log releases confusion

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#3461] [General]
There seems to be a version that was never released, neither finally nor as pre-release.

> [1.12.3 / 5.67.3] - 2023-11-
> Added
> 
>     [item]
> 
> Fixed
> 
>     [items]
> 
> [1.12.2 / 5.67.2] - 2023-11-
> Added
> [Items] [...]

Is there a reason to list the changes in two version numbers?

**Comment by DavidXanatos** [comment]
yes one release is today as pre release the other will be end of the week with newely signed driver

**Reply by bastik-1001** [reply]
Thank you for clearing up the confusion.
